src/odemis/gui/model/__init__.py

```diff
@@ -1173,6 +1174,7 @@ def moveStageBy(self, shift):
             kwargs["update"] = True
 
         f = self._stage.moveRel(move, **kwargs)
+        self.stage_is_moving.value = True
```

Comment:
Move the line _before_ starting the move, to be sure nothing thinks we are not moving while the stage position updates.